src/Application/Components/Shop/KeyBoundShopItem.jsx

```diff
@@ -27,12 +33,15 @@ class KeyBoundShopItem extends React.Component {
           </div>
           <div className="shop-item-number col-md-2">{this.props.owned}</div>
         </div>
+
```

Comment:
no newlines in html block, the `</div>` lines serve to visually chop it up well enough